changelog/unreleased/kong/11627.yaml

```diff
@@ -0,0 +1,7 @@
+message: Support observing the time consumed by some components in the given request.
+type: feature
+scope: Core
+prs:
```

Comment:
We should follow the rules in #11652, remove prs and jiras, and rename file to `per_request_debugging.yml`